benchmarks/benchmarks_suite_test.go

```diff
@@ -60,6 +60,7 @@ func init() {
 	if err != nil {
 		panic("Failed to marshal benchmark configuration file")
 	}
+	fmt.Printf("\nUsing test configuration:\n===============================\n%s\n", yamlFile)
```

Comment:
Can we drop verbose print outs here? If a user needs to see the static yaml file used, he/she can open it directly from the config folder.

Reply:
In the CI machine, we do not know the configuration, so this will print the configuration out and there will be no uncertainty on the configuration.  I do not care to drop but this kind of things are the things that increase productivity